src/gateway/router/routes/gcpAliveRoute.ts

```diff
@@ -1,5 +1,25 @@
 import Router from '@koa/router';
 
 export async function gcpAliveRoute(ctx: Router.RouterContext) {
-  ctx.body = 'ok';
+  ctx.body = {
+    'gateway': 'ok',
+    'db': await dbAccessible(ctx)
+  }
+}
+
+async function dbAccessible(ctx: Router.RouterContext) {
+  const { dbSource } = ctx;
+
+  try {
+    const result = await dbSource.healthCheck(`select max(id) as lastid from interactions`).timeout(500, { cancel: true });
```

Comment:
on the one hand...it's good that we have separate connection pool for a health check - as it will not affect the available connections in the main pool...

on the other hand...this check is not full representative...i.e. I wonder if it may happen, that the main connection pool does not work properly, whereas the 'helathCheck' connection pool works fine..